Source/Xamarin/Prism.Forms/Navigation/Xaml/NavigationExtensionBase.cs

```diff
@@ -44,6 +44,8 @@ protected internal get
             set => _sourcePage = value;
         }
 
+        public bool Animated { get; set; } = true;
+
```

Comment:
might as well add the UseModalNavigation property too :)

Reply:
Ok I will add it